test/integration/data/validation/test-report-webdriverIO.js

```diff
@@ -0,0 +1,99 @@
+export const testReportV2Partial = {
+	version: 2,
+	summary: {
+		status: 'failed',
+		framework: 'webdriverio',
+		count: { passed: 4, failed: 2, skipped: 2, flaky: 2 }
+	},
+	details: [{
+		name: 'reporter 1 > passed',
+		status: 'passed',
+		location: { file: 'test/integration/data/tests/webdriverIO/reporter-1.test.js' },
+		timeout: 2000,
+		tool: 'WebdriverIO 1 Test Reporting',
+		experience: 'Test Framework',
+		type: 'ui',
+		retries: 0
+	}, {
+		name: 'reporter 1 > skipped',
+		status: 'skipped',
+		location: { file: 'test/integration/data/tests/webdriverIO/reporter-1.test.js' },
+		timeout: 2000,
+		tool: 'WebdriverIO 1 Test Reporting',
+		experience: 'Test Framework',
+		type: 'ui',
+		retries: 0
+	}, {
+		name: 'reporter 1 > flaky',
+		status: 'passed',
+		location: { file: 'test/integration/data/tests/webdriverIO/reporter-1.test.js' },
+		timeout: 2000,
+		tool: 'WebdriverIO 1 Test Reporting',
+		experience: 'Test Framework',
+		type: 'ui',
+		retries: 2
+	}, {
+		name: 'reporter 1 > failed',
+		status: 'failed',
+		location: { file: 'test/integration/data/tests/webdriverIO/reporter-1.test.js' },
+		timeout: 2000,
+		tool: 'WebdriverIO 1 Test Reporting',
+		experience: 'Test Framework',
+		type: 'ui',
+		retries: 3
+	}, {
+		name: 'reporter 2 > passed',
+		status: 'passed',
+		location: { file: 'test/integration/data/tests/webdriverIO/reporter-2.test.js' },
+		timeout: 2000,
+		tool: 'Test Reporting',
+		experience: 'WebdriverIO 2 Test Framework',
+		type: 'integration',
+		retries: 0
+	}, {
+		name: 'reporter 2 > skipped',
+		status: 'skipped',
+		location: { file: 'test/integration/data/tests/webdriverIO/reporter-2.test.js' },
+		timeout: 2000,
+		tool: 'Test Reporting',
+		experience: 'WebdriverIO 2 Test Framework',
+		type: 'integration',
+		retries: 0
+	}, {
+		name: 'reporter 2 > flaky',
+		status: 'passed',
+		location: { file: 'test/integration/data/tests/webdriverIO/reporter-2.test.js' },
+		timeout: 2000,
+		tool: 'Test Reporting',
+		experience: 'WebdriverIO 2 Test Framework',
+		type: 'integration',
+		retries: 2
+	}, {
+		name: 'reporter 2 > failed',
+		status: 'failed',
+		location: { file: 'test/integration/data/tests/webdriverIO/reporter-2.test.js' },
+		timeout: 2000,
+		tool: 'Test Reporting',
+		experience: 'WebdriverIO 2 Test Framework',
+		type: 'integration',
+		retries: 3
+	}, {
+		name: 'reporter 1 > special/characters "(\\n\\r\\t\\b\\f)"',
+		status: 'passed',
+		location: { file: 'test/integration/data/tests/webdriverIO/reporter-1.test.js' },
+		timeout: 2000,
+		tool: 'WebdriverIO 1 Test Reporting',
+		experience: 'Test Framework',
+		type: 'ui',
+		retries: 0
+	}, {
+		name: 'reporter 2 > special/characters "(\\n\\r\\t\\b\\f)"',
+		status: 'passed',
+		location: { file: 'test/integration/data/tests/webdriverIO/reporter-2.test.js' },
+		timeout: 2000,
+		tool: 'Test Reporting',
+		experience: 'WebdriverIO 2 Test Framework',
```

Comment:
Oh this reminds me - you'll want to update the d2l-test-reporting.config.json for this repo too, to include the webdriver.io tests
https://github.com/Brightspace/test-reporting-node/blob/main/d2l-test-reporting.config.json